ebml_matroska.xml

```diff
@@ -431,9 +451,15 @@ the overlay track **SHOULD** be used instead. The order of multiple TrackOverlay
 If not found it **SHOULD** be the second, etc.</documentation>
     <extension webm="0"/>
   </element>
+  <element name="TrackLink" path="\Segment\Tracks\TrackEntry\TrackLink" id="0x6FAC" type="uinteger">
+    <documentation lang="en" purpose="definition">Specify that subtitle track is suitable for use with the audio Track specified (in the u-integer).
```

Comment:
IMO that should be in a different PR as it may delay the rest of the mundane stuff. This is a completely new feature (or at some point we had it and removed it).

As it can be multiple you may group them in a `TrackLinks` element, ending up with `path="\Segment\Tracks\TrackEntry\TrackLinks\TrackLink"`.

It adds more complexity in muxers as it forces them to recompute this mapping for every muxing (unless the track layout is untouched in the remuxing).

It may apply to audio as well. For example you may have 2 angles of the same scene in different rooms with different audio. The audio should match the room it corresponds to.

Reply:
Sure, I'll pull this out for now and we can discuss it in more detail in a separate PR after this one's all resolved.